### PR TITLE
Fixed the Web API docs

### PR DESCRIPTION
**Pull request by micheles**
job_type has been replaced by calculation_mode now.

### CODE REVIEW

File: doc/web-api.md [hunk 1]
```diff
@@ -18,19 +18,19 @@ Response:
     [{"description": "Hazard Calculation for end-to-end hazard+risk",
       "id": 1,
       "status": "executing",
-      "job_type": "hazard",
+      "calculation_mode": "classical",
       "is_running": true,
       "url": "http://localhost:8000/v1/calc/1"},
```
Comment (hascar) [inline doc/web-api.md]: The port is 8000 or 8800?

File: doc/web-api.md [hunk 2]
```diff
@@ -18,19 +18,19 @@ Response:
     [{"description": "Hazard Calculation for end-to-end hazard+risk",
       "id": 1,
       "status": "executing",
-      "job_type": "hazard",
+      "calculation_mode": "classical",
```
Comment (hascar) [inline doc/web-api.md]: I would do different examples: classical, event_based_risk, scenario or scenario_risk.

File: doc/web-api.md (hunk 1 above)
Comment (micheles) [inline doc/web-api.md]: 8800, I am changing all of the examples

File: doc/web-api.md (hunk 2 above)
Comment (micheles) [inline doc/web-api.md]: Done

File: doc/web-api.md [hunk 3]
```diff
@@ -18,21 +18,21 @@ Response:
     [{"description": "Hazard Calculation for end-to-end hazard+risk",
       "id": 1,
       "status": "executing",
```
Comment (hascar) [inline doc/web-api.md]: Missing owner

File: doc/web-api.md [hunk 4]
```diff
@@ -46,9 +46,9 @@ Response:
     {"description": "Hazard Calculation for end-to-end hazard+risk",
       "id": 1,
       "status": "executing",
-      "job_type": "hazard",
+      "calculation_mode": "classical",
       "is_running": true,
-      "url": "http://localhost:8000/v1/calc/1"}
+      "url": "http://localhost:8800/v1/calc/1"}
```
Comment (hascar) [inline doc/web-api.md]: Missing owner

**Comment by hascar** [comment]
In "GET /v1/calc/:calc_id/oqparam" they do not exist anymore these parameters: id, no_progress_timeout, sites and status

**Comment by micheles** [comment]
Removed id, no_progress_timeout and status; sites instead is can be present.

File: doc/web-api.md (hunk 3 above)
Comment (micheles) [inline doc/web-api.md]: Added

**Pull request merged**